src/xenia/vfs/virtual_file_system.cc

```diff
@@ -221,6 +222,26 @@ X_STATUS VirtualFileSystem::OpenFile(Entry* root_entry,
     if (entry->attributes() & kFileAttributeDirectory && is_non_directory) {
       return X_STATUS_FILE_IS_A_DIRECTORY;
     }
+
+    // If the entry does not exist on the host then remove the cached entry
+    // forcefully by "overwriting" the non-existent file
+    if (parent_entry) {
+      xe::vfs::HostPathEntry* host_Path =
```

Comment:
const + host_path instead of host_Path

Btw. What would happen if game tries to do something strange during savefile loading? It might expect different result?